### PR TITLE
Update Loki README to include links to modern docs

### PR DESCRIPTION
**Pull request by KMiller-Grafana**
Also fixed one reference to branch master, which is now main.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -32,15 +32,21 @@ Loki differs from Prometheus by focusing on logs instead of metrics, and deliver
...
+* [Getting Started](https://grafana.com/docs/loki/latest/getting-started/)
 
 ## Upgrading
 
 * [Upgrading Loki](https://grafana.com/docs/loki/latest/operations/upgrade/)
 
 ### Documentation
 
-* [master](https://grafana.com/docs/loki/latest/)
+* [v2.2.1](https://grafana.com/docs/loki/latest/)
```
Comment (sandeepsukhani) [inline README.md:43]: Should we link this to `v2.2.1` instead of `latest`?

**Review by sandeepsukhani — CHANGES_REQUESTED** [review]
Added some suggestions. Other than that, it looks good to me.

File: README.md (hunk 1 above)
Comment (dannykopping) [reply README.md:43]: I think we should link directly to `v2.2.1` in the docs and have separate links for `latest` and `next`.

Comment (KMiller-Grafana) [reply README.md:43]: Actually, given your comments, I'm going to close this PR.  I will introduce a new PR that revises what we present in the README such that a new release of Loki does not require an update to the README.  That'll reduce the effort of putting out a new release.

**Comment by KMiller-Grafana** [comment]
Closing PR.  Will open a new one with a revised README that does not require update upon each new release.

**Pull request closed without merging**